invoke-ai/InvokeAI
# feat(ui): various error toast optimizations

## Summary

* update socket invocation error toasts to not dismiss, but should be de-duplicated based on error type
* update copy if the error type is OOM
* add a sessionId that can easily be copied to clipboard 

## Related Issues / Discussions



## QA Instructions

Easiest way to test is to make a workflow that divides integers zero by zero. Queue up multiple and should only appear for the first one. If you dismiss toast before the queue is complete, another one should pop in its place.

## Merge Plan



## Checklist
